services/cartService.js

```diff
@@ -1,6 +1,30 @@
 const cartDao = require("../models/cartDao");
+const productDao = require("../models/productDao")
 const { validateQuantity, validateproductId } = require("../utils/validation");
 
+const deleteAllCart = async (userId) => {
+
+    const deleteAllCart = await cartDao.deleteAllCart(userId);
+    
+    return deleteAllCart;
+  };
+  
+const deleteCart = async (userId, productId) => {
+    validateproductId(productId);
+    
+    const AllProduct = await productDao.checkAllProduct()
```

Comment:
```suggestion
    const allProducts = await productDao.checkAllProduct()
```
네이밍이 클래스 같습니다! 문법 하이라이팅도 그렇게 되었네요 ㅋㅋㅋ